app/components/Admin/ApplicationReview/DecisionModal.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {Modal, Button, Alert} from 'react-bootstrap';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faEnvelope, faCheck, faTimes} from '@fortawesome/free-solid-svg-icons';
+import ModalSharedStyles from 'components/ModalSharedStyles';
+import {getUserFriendlyStatusLabel} from 'lib/text-transforms';
+
+interface Props {
+  show: boolean;
+  onHide: () => void;
+  onDecision: (decision: string) => void;
+  currentStatus: string;
```

Comment:
I had done that at one point, but was conflicted about separation of concerns as I'd be importing it from  `applicationReviewQuery.graphql` into a component that otherwise knows nothing about the application review page. 

As alternative I [specified the type in application-review](https://github.com/bcgov/cas-ciip-portal/pull/1644/files#diff-7bb8aa865fb388ebcf5c04ec0a682f26bfa9aa7480ec4e32a654e0a16ca089cdR134) where this value is fed into the mutation, and the status values are hard-coded into DecisionModal (eg: `onDecision('APPROVED')`).

On second thought the initial suggestion is probably not so bad, since it's just a type (only effects are pre-runtime) and not executable code. We should pick a way :)